### DISCUSSION TITLE
posts/xmas-security

### DISCUSSION THREAD

**Discussion by giscus** [whexy/whexy-blog-comments#12] [General]
# posts/xmas-security

https://www.whexy.com/posts/xmas-security

**Comment by MstMoonshine** [comment]
Very interesting!
Could you give a concrete example of a supple chain? What does the entire chain look like? And how is the structure of a supply chain related to/different from a software tech stack?

**Reply by whexy** [reply]
TBH I don't know much about the supple chain. I am going to read a lot about it in the future and write a more detailed post about it.
